examples/mobile/UIComponents/components/toast/index.html

```diff
@@ -53,6 +58,14 @@ <h1>
 					</button>
 				</div>
 			</div>
+			<div class="ui-popup ui-popup-toast ui-popup-toast-multiline" data-role="popup" id="popup_multiline_btn_toast">
+				<div class="ui-popup-content ui-popup-toast-has-button">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmo.
```

Comment:
euismod is a correct word.
Don't you know latin? ;)

Reply:
:)